### DISCUSSION TITLE
Talkにて未ログイン時にヘッダーのchatからsign in画面に遷移するとサーバーサイドでエラーが表示されます。

### DISCUSSION THREAD

**Discussion by TeruyaYuma** [sinProject-Inc/talk#1363] [General]
# 事象
未ログイン状態でヘッダーのchatからsign in画面に遷移するとcannot read propertyエラーが発生します。

# 再現方法
・末ログイン状態でヘッダーのchatからsign in画面に遷移
・ターミナル確認

※サーバーサイドで発生している為、コンソール、ブラウザ、画面では関連したエラーは表示されていません。
![スクリーンショット 2023-08-28 22 41 36](https://github.com/sinProject-Inc/talk/assets/55355758/b2d53ff2-30eb-404d-b1ba-5fec839fd3d7)

# 発生箇所
(authed)/chat/[room_id]/+page.server.ts

# 対応方法
上記ファイル6行目でlocalsのemailが取得できないのでエラーが発生しています。
emailがlocalsにない場合の分岐処理を入れることで対応できるかも（まだ前後処理など見れてないので断定できない要調査）

**Comment by iamsin** [comment]
@TeruyaYuma 

ご連絡ありがとうございます。Issues を作成して対応していきたいと思います。
